docs/users-guide/15-alerts.md

```diff
@@ -0,0 +1,71 @@
+
+## Getting alerts about questions
```

Comment:
we should also add this page to the index/readme of the users guide

Reply:
Yep, good catch.